Gopkg.toml

```diff
@@ -259,3 +259,7 @@
 [[constraint]]
   name = "github.com/twmb/murmur3"
   version = "1.0.0"
+
+[[constraint]]
+  branch = "master"
```

Comment:
If there aren't releases can we pin this to a specific rev?  Otherwise the builds won't be reproducible.